webapp/components/interop-dashboard.js

```diff
@@ -267,6 +278,12 @@ class InteropDashboard extends PolymerElement {
         }
 
       </style>
+      <div class="previous-year-banner" hidden$=[[isCurrentYear]]>
```

Comment:
Minor nit: isCurrentYear -> this default value for this should be true. I suspect that there might be a minor time gap when isCurrentYear is initialized and rendered here vs when ready() is called

Reply:
Good idea - I've set it to have a default value of `true`.